tests/phpunit/tests/functions/wpuploadbits.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Test the wp_upload_bits function
+ *
+ * @group Functions
+ * @group Upload
+ * @covers ::wp_upload_bits
+ */
+
+class tests_wp_upload_bits extends WP_UnitTestCase {
```

Comment:
```suggestion
class Tests_Functions_WpUploadBits extends WP_UnitTestCase {
```
- Test class names should follow the `Tests_Directory_ClassOrFunctionInTitleCase` format. See [Writing PHPUnit Tests - Test Classes](https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#test-classes).